### PR TITLE
Refactor locales

### PR DESCRIPTION
**Pull request by emmabeynon**
As a follow on to https://github.com/alphagov/collections/pull/1327 which introduced some inconsistency to the location of translation files, this PR reorganises the locales directory with the aim of splitting out translations into smaller, clearly named files which should make it easier for developers to add and update translations in future.  As a bonus, this also gets rid of any translations that were found to no longer be used in the app.

### CODE REVIEW

File: test/presenters/brexit_landing_page_presenter_test.rb [hunk 1]
```diff
@@ -109,8 +109,8 @@
 
   describe "#email_path" do
     it "strips the locale extension from the base path if present" do
-      I18n.with_locale(:fr) do
-        taxon = Taxon.new(ContentItem.new("base_path" => "/base_path.fr"))
+      I18n.with_locale(:cy) do
```
[the remark this answers is not in the document]
Comment (emmabeynon) [reply test/presenters/brexit_landing_page_presenter_test.rb]: After changing from loading the campaign buckets as a YAML file to using I18n (https://github.com/alphagov/collections/pull/1346/commits/3ea3c92bfa71fb145acd1a8ac413bde4b04a0f3e#diff-e85c65799e3ed5ade04dee5e8ed37f93R54) this test broke as it wouldn't return an array since `fr` translation files don't exist in this repo, which meant that an error occurred when trying to iterate over it.

**Comment by emmabeynon** [comment]
@benthorner Thanks for reviewing.  I wasn't aware there was a Rails convention of keeping locale/config mashup files separate, though to be honest I've not seen that before!  I'm struggling to find any guidance about the best way of working with such an approach which in itself suggests its pretty atypical.  I suppose my concern with keeping the buckets out of the locales directory is that it wasn't obvious to me on first glance why we would do that (again, probably because it's atypical).  Perhaps a compromise would be to have the bucket translations in their own subdirectory under locales, which would have the benefit of not needing to make the changes I made (here) [https://github.com/alphagov/collections/pull/1346/commits/3ea3c92bfa71fb145acd1a8ac413bde4b04a0f3e#diff-e85c65799e3ed5ade04dee5e8ed37f93R54] as we can still load it as a YAML file?  What do you think?

File: test/presenters/brexit_landing_page_presenter_test.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (benthorner) [reply test/presenters/brexit_landing_page_presenter_test.rb]: Ah yes, I see. That should be fine - just trying to think if there's a new edge case there.
